webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx

```diff
@@ -109,6 +109,8 @@ const AdvancedTextEditor = ({
 
     const currentUserId = useSelector(getCurrentUserId);
     const channelDisplayName = useSelector((state: GlobalState) => getChannelSelector(state, channelId)?.display_name || '');
+    const channelType = useSelector((state: GlobalState) => getChannelSelector(state, channelId)?.type || '');
+    const isChannelShared = useSelector((state: GlobalState) => getChannelSelector(state, channelId)?.shared);
```

Comment:
One thing...should we just call `getChannelSelector` once?

Reply:
+1

Reply:
Fixed on 04f11e1, PTAL